### PR TITLE
ci(swiftlint): add in swiftlinter

### PR DESCRIPTION
**Pull request by bassrock**
## Summary
* Follow the mozila ios engineering swiftlint rules that are being adopted.

## Implementation Details
* Update the build phase to use the swift linter. 
* Will need to update the bitrise once https://github.com/Pocket/pocket-ios/pull/231 is merged to download swiftlint

## Test Steps
* Should not regress. No functionality or functional code changes made

## PR Checklist:
- [x] Added Unit / UI tests
- [x] Self Review (review, clean up, documentation, run tests)
- [x] Basic Self QA

### CODE REVIEW

**Review by cyndichin — COMMENTED** [review]
thanks Daniel! as mentioned in a couple of other PRs, we may want to add https://realm.github.io/SwiftLint/multiline_arguments_brackets.html
I'll check with Firefox team to see what they think too

https://github.com/Pocket/pocket-ios/pull/227#discussion_r954098412

**Comment by bassrock** [comment]
@cyndichin ready for review, and added a CI step for the xcode build.

What do you think about merging this and then doing incremental updates when we align on more rules with the rest of the Mozilla org?

File: bitrise.yml [hunk 1]
```diff
@@ -10,6 +10,9 @@ trigger_map:
 workflows:
   app-store-build:
     steps:
+      - brew-install@0:
```
Comment (bassrock) [inline bitrise.yml:13]: Adding swiftlint to the bitrise file.

File: setup_build_tools.sh [hunk 2]
```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+#
+# Install all the required dependencies for building and deploying Pocket for iOS
```
Comment (bassrock) [inline setup_build_tools.sh:4]: A helper script to help new devs install required tools to build pocket.

**Review by jacobsimeon — CHANGES_REQUESTED** [review]
I'd love to have a conversation about this as team before we go ahead with the merge.

**Review by jacobsimeon — APPROVED** [review]

**Pull request merged**